### PR TITLE
Data class copy

### PR DESCRIPTION
**Pull request by wisnup**

### CODE REVIEW

File: Classes/Data class copy/src/Task.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+data class Person(val name: String, val age: Int)
+
+fun oneYearOlder(person: Person): Person {
+    return person.copy(age = person.age + 1)
+}
```
Comment (leo0263) [inline Classes/Data class copy/src/Task.kt]: Please enable `Ensure every saved file ends with a line break` options.

File: Classes/Data class copy/task-info.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+type: edu
+files:
+- name: src/Task.kt
+  visible: true
+  placeholders:
+  - offset: 105
+    length: 33
+    placeholder_text: TODO()
+- name: test/Tests.kt
+  visible: true
```
Comment (leo0263) [inline Classes/Data class copy/task-info.yaml]: Is the test intentionally set to be visible?

File: Classes/Data class copy/test/Tests.kt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+    @Test
+    fun testSolution() {
+        val person = Person("Quippo", 10)
+        val olderPerson = oneYearOlder(person)
+
+        assertEquals(person.name, olderPerson.name)
+        assertEquals(11, olderPerson.age)
+    }
+}
```
Comment (leo0263) [inline Classes/Data class copy/test/Tests.kt]: New line please.

**Review by leo0263 — CHANGES_REQUESTED** [review]
Tested & working, just a few nitpicks. Please check 🙏

**Comment by wisnup** [comment]
@leo0263 code updated, please check

File: Classes/Data class copy/task-info.yaml (hunk 2 above)
Comment (wisnup) [reply Classes/Data class copy/task-info.yaml]: no

**Review by leo0263 — APPROVED** [review]
LGTM and tested 👍

**Pull request merged**